controllers/ncnews_controllers.js

```diff
@@ -17,3 +21,18 @@ selectArticleById(articleId).then((article)=>{
 })
 .catch(next)
 }
+
+exports.patchArticle = (request, response, next) => {
+// access user articleId and request body
+const {article_id: articleId} = request.params;
+const vote = request.body;
+// invoke model function with articleId and requestBody
+updateArticleById(articleId, vote).then((article)=>{
+//respond back to client with updated article 
+    response.status(201).send({article:article })
+})
+.catch(next)
+}
+
+
+
```

Comment:
Please rename your controllers and models to reflect the section of the API you're creating aka you should have a 'articles.controllers.js' and a 'topics.controllers.js' and a 'users.controllers.js' so far